### PR TITLE
Handle negative OFFSET and LIMIT

### PR DESCRIPTION
**Pull request by msullivan**
Nothing important here, but might as well not produce an ISE.

### CODE REVIEW

File: edb/api/errors.txt [hunk 1]
```diff
@@ -112,6 +112,8 @@
 0x_05_01_00_00   InvalidValueError
 0x_05_01_00_01   DivisionByZeroError
 0x_05_01_00_02   NumericOutOfRangeError
+0x_05_01_00_03   InvalidLimitError
```
Comment (msullivan) [inline edb/api/errors.txt:115]: Is it worth adding new error types for this? We could just use InvalidValueError directly? (Does adding error types require updating the bindings?)

Comment (elprans) [reply edb/api/errors.txt:115]: Probably not, because we are mulling possibly adding support for negative `LIMIT`/`OFFSET` (see #211).

Comment (msullivan) [reply edb/api/errors.txt:115]: Alright, so just make it an `InvalidValueError`?

Comment (elprans) [reply edb/api/errors.txt]: Yeah, that seems reasonable.

**Review by elprans — APPROVED** [review]

**Comment by msullivan** [comment]
Going to cherry-pick this since it is wildly low risk

**Pull request merged**